src/api/spec/requests/kerberos_login_spec.rb

```diff
@@ -1,82 +1,64 @@
 require 'rails_helper'
-
+require 'authenticator'
```

Comment:
It would be ideal if we could get rid of this require since request specs are supposed to be integration tests so the only things that should be mocked are external services like GSSApi.